client/webserver/site/src/js/locales.ts

```diff
@@ -224,7 +241,24 @@ export const enUS: Locale = {
   [ID_CONNECT_WALLET_ERR_MSG]: 'Failed to connect {{ assetName }} wallet: {{ errMsg }}',
   [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'Redeemed by {{ makerAddr }}',
   [ID_REFUND_IMMINENT]: 'Will happen in the next few blocks',
-  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Refund will happen after {{ refundAfterTime }}'
+  [ID_REFUND_WILL_HAPPEN_AFTER]: 'Refund will happen after {{ refundAfterTime }}',
+  [ID_AVAILABLE_TITLE]: 'Available',
+  [ID_LOCKED_TITLE]: 'Locked',
+  [ID_IMMATURE_TITLE]: 'Immature',
+  [ID_SWAPPING]: 'Swapping',
+  [ID_BONDED]: 'Bonded',
+  [ID_LOCKED_BAL_MSG]: 'Total funds temporarily locked to cover the costs of your bond maintenance, settling matches, and necessary fees',
```

Comment:
~~settling matches, and necessary fees~~ -> live orders and matches

Also can be locked for other reasons though. For Decred, we add value locked in tickets too.

Reply:
Updated.